<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authorization</class>
 <widget class="QMainWindow" name="Authorization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>668</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>665</width>
    <height>668</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>665</width>
    <height>668</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Authorization</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>551</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/dada.png</pixmap>
    </property>
   </widget>
   <widget class="QTabWidget" name="AuthorizationTabWidjet">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>571</width>
      <height>451</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>571</width>
      <height>451</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>571</width>
      <height>451</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
QTabBar
{
background-color: #0B6C87;
color:#ffffff;
}
QTabBar::tab:!selected 
{
height: 60px;
 width: 125px;
background-color: #0B6C87; 
color:#ffffff;
font-size:11pt;
}
QTabBar::tab:selected {
height: 60px; 
width: 125px;
background-color: #0BAFF2; 
color:#ffffff;
font-size:12pt; 
font-weight: bold;
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>16</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Вход</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="loginLabel">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Логин:</string>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="loginLineEdit">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="passwordLabel">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Пароль:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="EnteringPushButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 12px;</string>
        </property>
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Авторизация</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="loginLabel_2">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Логин:</string>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="loginLineEdit_2"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="passwordLabel_2">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Пароль:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLineEdit_2">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="EnteringPushButton_2">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 12px;</string>
        </property>
        <property name="text">
         <string>Создать</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
